python/lsst/ts/mtmount/mtmount_csc.py

```diff
@@ -843,30 +852,87 @@ async def send_command(self, command, do_lock):
             commands can run simultaneously, so specify True when practical.
             Specify False for stop commands and the camera cable wrap
             tracking command (so rotator following is not blocked).
+        timeout : `float` | `None`
+            Timeout for initial ack.
+            If None then use self.config.ack_timeout.
 
         Returns
         -------
         command_futures : `CommandFutures`
             Futures that monitor the command.
+
+        Raises
+        ------
+        asyncio.TimeoutError
+            If the timeout is exceeded.
+        salobj.ExpectedError
+            If not connected to the low-level controller.
+        RuntimeError
+            If the sequence_id is already in use.
+            This indicates an internal error.
         """
         if do_lock:
             async with self.command_lock:
-                return await self._basic_send_command(command=command)
+                return await self._basic_send_command(command=command, timeout=timeout)
         else:
-            return await self._basic_send_command(command=command)
+            return await self._basic_send_command(command=command, timeout=timeout)
+
+    async def retry_command(
+        self, command, *, max_tries=3, timeout=SHORT_COMMAND_TIMEOUT
+    ):
+        """Send a low-level command, retrying up to max_tries times.
+
+        Parameters
+        ----------
+        command : `commands.Command`
+            Class of command to send
+        max_tries : `int`
+            Maximum number of times to issue the command.
+        timeout : `float`
+            Timeout (sec) for the command (applied to each retry).
+            Keep it short to avoid blocking the command lock for a long time.
```

Comment:
It looks like the `retry_command` command is only used internally to this CSC class and it never gets called with a specific value for `timeout`. Is this argument added for future use or with some specific use in mind that I fail to notice?

Reply:
I wanted the flexibility to call it with different timeouts. But at the moment I have no use case for that.